kubernetes/helm_charts/monitoring/dashboards/dashboards/nginx.json

```diff
@@ -66,7 +74,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(nginx_http_requests_total{cluster=~\"$cluster\"}[2m]))",
+          "expr": "sum(rate(nginx_http_requests_total{sb_cluster=~\"$cluster\"}[2m]))",
```

Comment:
How will we handle historical data? Probably we should release these as beta dashboard and change to main one after a couple of days? Or should have both cluster in query

Reply:
there are 2 entries, one with cluster and one with sb_cluster. so that we'll have both